### ISSUE TITLE
ClojureScript warning: var is not public

### ISSUE THREAD

**Issue by miikka** [bhb/expound#137]
The new 1.10.439 release of ClojureScript now [gives warnings when you use private vars](https://clojurescript.org/news/news#_warnings_on_private_var_use). I'm using Expound 0.7.1 with Orchestra 2018.11.07-1 and I'm now seeing the following warnings in my project when I call `orchestra-cljs.spec.test/instrument`:

```
WARNING: var: expound.alpha/specs is not public at line 27 src/x.cljs
WARNING: var: expound.alpha/value-in-context is not public at line 27 src/x.cljs
```

**Comment by Odinodin** [comment]
Just hit the exact same issue.

**Comment by bhb** [comment]
Thanks for reporting this! I’ll look into it.

**Comment by bhb** [comment]
Repro:

`rlwrap clojure -Sdeps '{:deps {orchestra {:mvn/version "2018.11.07-1"} expound {:mvn/version "0.7.1"} org.clojure/clojurescript {:mvn/version "1.10.439"}}}' --main cljs.main`

```
ClojureScript 1.10.439
cljs.user=> (require '[orchestra-cljs.spec.test])
nil
cljs.user=> (require '[expound.alpha :as expound])
nil
cljs.user=> (orchestra-cljs.spec.test/instrument)
WARNING: var: expound.alpha/specs is not public at line 1 <cljs repl>
WARNING: var: expound.alpha/value-in-context is not public at line 1 <cljs repl>
[expound.printer/indent expound.paths/prefix-path? expound.alpha/explain-result-str expound.alpha/specs expound.alpha/explain-results expound.alpha/error-message expound.alpha/custom-printer expound.paths/kvps-path? expound.alpha/explain-results-str expound.alpha/value-in-context expound.printer/pprint-str expound.alpha/expound expound.alpha/defmsg expound.alpha/expound-str expound.paths/kps-path? expound.alpha/printer expound.alpha/explain-result expound.problems/summary-form expound.printer/no-trailing-whitespace]
cljs.user=>
```

**Comment by bhb** [comment]
Thanks again for reporting this! This appears to be an issue in Orchestra, not Expound:

Start a REPL with

`rlwrap clojure -Sdeps '{:deps {org.clojure/test.check {:mvn/version "0.9.0"}  orchestra {:mvn/version "2018.11.07-1"} org.clojure/clojurescript {:mvn/version "1.10.439"}}}' --main cljs.main`

```
(in-ns 'ex.one)
(require '[cljs.spec.alpha :as s])
(s/fdef foobar :args (s/cat :x int?))
(defn- foobar [x] x)
(in-ns 'ex.two)
(require '[orchestra-cljs.spec.test :as orch.st])
(orch.st/instrument)
```

Expected: Output should be `[ex.one/foobar]`
Actual: Output is:

```
WARNING: var: ex.one/foobar is not public at line 1 <cljs repl>
[ex.one/foobar]
```

Happily, it looks like @jeaye has already fixed this issue in master - as of orchestra 09fb53f2f9b097984ceb68d5b6c039b2d9a2ec8e, this no longer reproduces for me, so this fix should be in the next release 😄
